airbyte-integrations/connectors/destination-postgres/src/integrationTestLegacy/kotlin/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresRawOverrideDisableTypingDedupingTest.kt

```diff
@@ -24,4 +27,27 @@ class PostgresRawOverrideDisableTypingDedupingTest : PostgresTypingDedupingTest(
     @Disabled @Test override fun identicalNameSimultaneousSync() {}
 
     @Disabled @Test override fun testVarcharLimitOver64K() {}
+
+    //syncs used to fail when doing this, not anymore
+    @Disabled @Test override fun interruptedTruncateWithPriorData() {}
+
+    // fields that are not in the schema are now dropped.
+    @Disabled @ParameterizedTest @ValueSource(longs = [0L, 42L]) override fun testIncrementalSyncDropOneColumn(inputGenerationId: Long) {}
+
+    // this tests that the fully qualified raw table name is lowercased. This is actually not a restriction anymore, so disabling the test.
+    @Disabled @Test override fun testMixedCasedSchema() {}
```

Comment:
https://docs.airbyte.com/integrations/destinations/postgres#naming-conventions

> Airbyte Postgres destination will create raw tables and schemas using the Unquoted identifiers by replacing any special characters with an underscore. All final tables and their corresponding columns are created using Quoted identifiers preserving the case sensitivity. Special characters in final tables are replaced with underscores

https://github.com/airbytehq/airbyte/pull/35042/files

I'm not sure why we had that restriction for raw tables, but we don't have this restriction anymore